src/main/java/io/openliberty/tools/common/plugins/util/DevUtil.java

```diff
@@ -3877,6 +3936,18 @@ private ProjectModule getFirstProjectModule(ProjectModule project, File buildFil
         }
         return null;
     }
+    
+    protected void resourceDirectoryCreated() throws IOException {
+
+	}
+
+	protected void resourceModifiedOrCreated(File fileChanged, File resourceParent, File outputDirectory) throws IOException {
+        copyFile(fileChanged, resourceParent, outputDirectory, null);
+	}
+
+    protected void resourceDeleted(File fileChanged, File resourceParent, File outputDirectory) throws IOException {
+        deleteFile(fileChanged, resourceParent, outputDirectory, null);
+	}
```

Comment:
Indentation is off here as well

Reply:
So these methods are overriden in ci.maven, meaning this default implementation is only used with cigradle.  While there is nothing wrong with this, I'd prefer to make these `abstract` and override/implement them separately in ci.maven and ci.gradle to avoid confusion.

Reply:
Done